lib/engine/round/auction.rb

```diff
@@ -59,8 +59,10 @@ def pass_description
 
       def process_action(action)
         clear_cache!
+
         step = @steps.find { |s| s.actions(action.entity).include?(action.type) }
         raise GameError, "No step found for this action: #{action.to_h}" unless step
+        raise GameError, "It is not #{action.entity.name}'s turn" unless step.active_entities.include?(action.entity)
```

Comment:
how is this possible? there should be no step available if it is not that user's turn

Reply:
yep, your quite right, i hadn't understood about the entity parameter